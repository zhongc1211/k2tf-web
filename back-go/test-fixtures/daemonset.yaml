apiVersion: apps/v1
kind: DaemonSet
metadata:
  labels:
    test: MyExampleApp
  name: terraform-example
  namespace: something
spec:
  selector:
    matchLabels:
      test: MyExampleApp
  template:
    metadata:
      labels:
        test: MyExampleApp
    spec:
      containers:
        - image: nginx:1.7.8
          name: example

          resources:
            limits:
              memory: 512Mi
            requests:
              cpu: 250m
