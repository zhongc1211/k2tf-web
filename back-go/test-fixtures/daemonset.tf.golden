resource "kubernetes_daemonset" "terraform_example" {
  metadata {
    name      = "terraform-example"
    namespace = "something"
    labels    = { test = "MyExampleApp" }
  }
  spec {
    selector {
      match_labels = { test = "MyExampleApp" }
    }
    template {
      metadata {
        labels = { test = "MyExampleApp" }
      }
      spec {
        container {
          name  = "example"
          image = "nginx:1.7.8"
          resources {
            limits   = { memory = "512Mi" }
            requests = { cpu = "250m" }
          }
        }
      }
    }
  }
}
